### PR TITLE
Create 2016-07-24-top-fashion-accessories-for-men.markdown

### PR DESCRIPTION
**Pull request by fasih**
Added file on behalf of @aditigupta1527

### CODE REVIEW

**Comment by fasih** [comment]
@smritisharmaa Please review this article.

**Comment by smritisharmaa** [comment]
Sure will review and update you till tomorrow afternoon

**Comment by fasih** [comment]
Is review done @smritisharmaa? For images @aditigupta1527 you have to upload it by drag and drop in PR edit section.

**Comment by smritisharmaa** [comment]
Hey Fasih...

I am unable to view the full article. Half of it I saw need some changes
but I cannot view the whole article. Kindly tell me how to and also.. I
will send u morw articles tomorrow

**Comment by fasih** [comment]
@smritisharmaa link to the actual article file 
https://github.com/junkjam/cms.blog.junkjam.in/pull/22/files. 
Just open it in a web browser and click plus icon on the line where you want to add a comment.

**Comment by aditigupta1527** [comment]
watch
ties
money_clip
jewelry
card_holder
belt
cuff_link
pocket_square
sunglasses
wallet

File: source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Top Fashion Accessories for Men
+
+Most of the people will concur with the epigram "women's fashion is much more diverse than men!!". But who says accessories are for women only? Here are top 10 accessories that can add flavour and color to any outfit to fit the preferences of any men out there.
+
+## 1. Watch
+A wrist watch is the most conventional accessory a man can invest in. it adds a punch of simplicity as well as style to your personality. it is one of the most acceptable men's accessory. Mechanical, chronograph, leather strap watches are the latest trends this season.
```
Comment (smritisharmaa) [inline source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown]: In the second line 
it (with a capital I) and remove the word punch, add another word synonymous and relevant to it

File: source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## 2. Tie
+A tie is pretty much one accessory that adds life to any men's costume. Various mutants include bow-tie, necktie, ascots. A tie radiates class and confidence. Pair up your suit with multicolor, printed , textured, silk ties this summer.
```
Comment (smritisharmaa) [inline source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown]: A tie is pretty much one accessory that adds life to any men's costume. (incorrect grammatically, please reframe the line)

File: source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## 7. Jewellery for men
+Yes unsual it seems,, but jewellery for men is gaining popularity now-a-days. Common jewellery for men is the wedding ring. THe most effective way for a man to carry jewellery is to be comfortable about it. Ear studs, copper bracelets, lapel pins, beaded necklace are some well executed jewellery intended for men.
+
+## 8. Belt
+Belt is a safe and easy pick accessory for men. It no longer serves aa s tool for holding mne's trouser but a great style statement. Wide buckles and a webbed, fabric version with a double-D buckle fastening works well for this summer.
+
+## 9. Sunglasses
+Sunglasses, aviators, shades are one of the accessory men wear on their face. They are a must have for glamorous, High Fashion look. Tortoiseshell frames and sleek metal are latest fashion trends..
+
+## 10. Cuff links
```
Comment (smritisharmaa) [inline source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown:32]: The article seems to be great and well-researched.
But the author must know that simply decorating the article with heavy words does not mean your article is good, you must add relevant words and such words which are easily understandable by the common audience. Nobody wants to read an article which makes them think twice about words used in it.
You have a good vocabulary but you need to understand the relevancy and make it more human interesting. Although, it is a very good attempt.

**Comment by smritisharmaa** [comment]
Hey,

I have added my comments on the article. Kindly ask the author to review it
and make the changes

**Comment by fasih** [comment]
Thanks a ton @smritisharmaa. Suggested changes has been updated in this PR by @aditigupta1527 .

**Comment by fasih** [comment]
Added pic for facebook og image and listing image of this article.
menfashionaccessories

**Comment by fasih** [comment]
Article has been published @aditigupta1527 
URL : https://blog.junkjam.in/2016/07/24/top-fashion-accessories-for-men/

**Comment by smritisharmaa** [comment]
Hey Fasih,

No update on other articles that I sent you?

**Comment by fasih** [comment]
Hi @smritisharmaa 
I haven't received any article from you except first one. Your first post issue and PR #18 respectively.

**Pull request merged**